test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -168,6 +170,37 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			}
 		})
 
+		It("should validate Ubuntu host OS network configuration on all nodes", func() {
+			if eng.ExpandedDefinition.Properties.IsUbuntuDistroForAllNodes() {
+				kubeConfig, err := GetConfig()
```

Comment:
We can clean up some of this boilerplate in the future, but this is (IMO) O.K. as a pattern we can copy/paste develop for the remaining CIS changes